data/incipias/incipias first contact.txt

```diff
@@ -0,0 +1,405 @@
+				goto choice
+			label yes
+			`	"Happy I am that you agree to that condition. Bear in mind, Captain, that the slightest misstep may have grave consequences." The Quarg stares you down. "We will be watching your actions, to verify that they are peaceful and without ill or unwise intent. Please for a moment remain here," it says, and walks away.`
+			`	The Quarg only takes a moment to come back; it is now carrying a sleek silver box with a display on one side.`
+			`	"This is our translator, our link to the Incipias. It is able to translate their unwonted language into your speech as well. Be aware of the honor of me gifting this to you," it says.`
+			`	As you stow it away, the Quarg speaks once more. "To satisfactorily utilize this device, you must direct it towards the source of the communication, so that it is made able to discern the relevant emissions. Given the complexity of their language, the translator might not be able to adequately translate some aspects of their obscure speech."`
+			`	It then looks you directly in the eyes. "Remember what I have told you, human. The stars have no place for those who would act foolishly and thoughtlessly towards their fellow beings." The Quarg then walks away, leaving you alone.`
+				accept
+			label declined
+			`	"We are gracious for your understanding, but remember thus: the Incipias are under our watch and care. None shall come to harm them without incurring our wrath." With these words the Quarg turns away.`
+				decline
+	on enter Dom'us
+		"reputation: Hicemus" += 10
+		conversation
+			`Entering the system with the translator, you prepare to broadcast your first message.`
+			choice
+				`	"Hello. I am Captain <first> <last>, here to make contact."`
+				`	"Greetings, strangers. I seek to help you."`
+				`	"I come in peace."`
+			`	You notice that your translator starts to flash a sequence of lights, and you point it towards your ship's camera. For a minute, nothing happens at all.`
+			`	After some time, a wave of movement passes through all ships in the system, and a communication channel opens from the space station. Colors rush over your display: a mix of yellow and green. You point your translator towards the flashing lights and hope it is able to pick up whatever these aliens are saying to you.`
+			`	"Welcome world-traveler. We greet you in our home-space." The voice is smooth and synthetic, lacking in any discernible emotion.`
+			`	The colors stop before rushing over your screen again.`
+			`	"What is your intention-goal here?"`
+			choice
+				`	"I am here to make peaceful contact with you."`
+				`	"I want to help one of your stranded ships."`
+			`	Again a long pause. "Your intention-goals are welcome. Please dock to the capital-station."`
+			`	Along the message comes a small map of the station with the docking ports highlighted.`
```

Comment:
```suggestion
				`	"I want to help one of your stranded ships."`
			`	Again a long pause. "Your intention-goals are welcome. Please dock to the capital-station." Along the message comes a small map of the station with the docking ports highlighted.`
```
Ditto; unless a choice is splitting up short sentences, there's no need to do it otherwise.